src/helpers/FormatTemperature.ts

```diff
@@ -0,0 +1,4 @@
+export default function formatTemperature (temperature?: number) : string{
+    return temperature ? Math.round(temperature) + ' °C' : "N/A";
+}
+
```

Comment:
Could we add a couple of tests for this?

Reply:
Shouldn't be an issue, probably a new ticket though? since this is all under CSS refactor...?

Reply:
This is the PR where we've created this function, so we should really test it at the same time.